subprojects/problems/src/integTest/groovy/org/gradle/api/problems/ProblemsServiceIntegrationTest.groovy

```diff
@@ -45,7 +45,7 @@ class ProblemsServiceIntegrationTest extends AbstractIntegrationSpec {
                         it.label("label")
                         .undocumented()
                         .noLocation()
-                        .type("type")
+                        .category("type")
```

Comment:
❌ The assertion on line 60 should look like this, shouldn't it?
`this.collectedProblems[0]["category"] == "type"`